js/controller.js

```diff
@@ -604,13 +612,17 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
-    onFullscreenChanged: function(event, fullscreen, paused) {
+    onFullscreenChanged: function() {
+      if (this.state.isFullScreenSupported) {
```

Comment:
I'm getting this error - 

Failed to execute 'requestFullScreen' on 'Element': API can only be initiated by a user gesture.